lib/autoprefixer-rails/sprockets.rb

```diff
@@ -57,7 +57,7 @@ def initialize(filename)
     end
 
     # Sprockets 2 API new and render
-    def render(_, _)
+    def render(_a, _b)
```

Comment:
Generally, I'm not inclined to accept changes to workaround bugs in 3rd party tools that we're not using.
However, if changing this line to `render(*)` could solve your problem, I'll accept it.

Reply:
Thanks @albb0920, I thought it was worth a shot doing the PR since the AWS gem accepted a similar one: https://github.com/aws/aws-sdk-ruby/pull/2426/files, which was improved upon in https://github.com/aws/aws-sdk-ruby/pull/2428/files. However, I understand that you have your own set of requirements accepting PRs. I'm not sure about the `*`, would rather wait to see if Sorbet sorts it out on their end.

Reply:
Just commenting to let know that this would really help us too, tho I get it that its not properly related to this gem. Please consider accepting this in the meanwhile.

Reply:
I just tested this and `*` works… Could we get this PR updated to use this?

```suggestion
    def render(*)
```